docs/env_configuration.md

```diff
@@ -1,14 +1,18 @@
+## Shared Configuration
+
+Force uses [shared configuration](https://github.com/artsy/README/blob/main/playbooks/development-environments.md#shared-configuration) to distribute common and sensitive configuration values. The [setup script](../scripts/setup.sh) will download `.shared.env` and also initialize a `.env` (from `.env.example`) for developer custom configuration and any overrides.
+
 ## Adding an ENV var
 
 ### Server side only
 
-1. Add the variable to your `.env` file.
+1. Add the variable to `.shared.env` file and update the S3 version.
```

Comment:
(Looks like suggest edit button went away)

Even though the shared config link is just above, it'd still be helpful to link it here to, so

```
1. Add the variable to `.shared.env` file and [update the S3 version](https://github.com/artsy/README/blob/main/playbooks/development-environments.md#shared-configuration).
```

(Side note -- very much share the same concern [as @jonallured here](https://artsy.slack.com/archives/CP9P4KR35/p1626960318203300?thread_ts=1626932540.200100&cid=CP9P4KR35) about these values not living in version control. At the very least it'd be interesting to see historical values as they change through time. ENVs very much tell a story.)

Reply:
I added the links.

Will think on / start a discussion about how we should handle tracking of ENVs over time. Preferably the process should work for both, private and OSS projects that use this `.env.shared` configuration approach.